src/modules/APImethods.js

```diff
@@ -0,0 +1,30 @@
+const tvmazeApiBaseUrl = 'https://api.tvmaze.com/lookup/shows?imdb=';
+const involvementBaseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/dnotAWJTR3nbocRUoECT/';
+
+/* eslint-disable no-await-in-loop */
+export const getShowsData = async (showsIds) => {
+  const showsData = [];
+  for (let i = 0; i < (showsIds.length); i += 1) {
+    const data = await fetch(tvmazeApiBaseUrl + showsIds[i]).then((data) => data.json());
+    showsData.push(data);
+  }
```

Comment:
**[Mandatory Requirement]**
- [ ] Please use forEach instead of for loop to follow JavaScript best practices :+1: